spec/requests/home_spec.rb

```diff
@@ -6,6 +6,10 @@
   end
 
   context 'without a current tenant' do
+    before do
+      allow(ApplicatioController).to receive(:multitenant?).and_return(true)
```

Comment:
`s/ApplicatioController/ApplicationController/`

Also, you're using `#multitenant?` here and `#multitenant` (sans `?`) above on line 19 of the `ApplicationController`.